README.md

```diff
@@ -35,23 +35,23 @@ Node.js >= 6.0.0 required.
 
 ## Features
 
-- ✔︎ Build-in process management
+- ✔︎ Built-in process management
 - ✔︎ Plugin system
 - ✔︎ Framework customization
 - ✔︎ Lots of [plugins](https://eggjs.org/badgeboard/)
```

Comment:
help to replace plugin url with `https://github.com/search?q=topic%3Aegg-plugin&type=Repositories`